src/_P076_HLW8012.ino

```diff
@@ -386,18 +440,41 @@ boolean Plugin_076(uint8_t function, struct EventStruct *event, String& string)
             success = true;
           }
 
-          UserVar.setFloat(event->TaskIndex, 0, p076_hvoltage);
-          UserVar.setFloat(event->TaskIndex, 1, p076_hcurrent);
-          UserVar.setFloat(event->TaskIndex, 2, p076_hpower);
-          UserVar.setFloat(event->TaskIndex, 3, p076_hpowfact);
+          p076_hpowerReactive = Plugin_076_hlw->getReactivePower(valid);
+
+          if (valid) {
+            success = true;
+          }
+
+          p076_hpowerApparent = Plugin_076_hlw->getApparentPower(valid);
+
+          if (valid) {
+            success = true;
```

Comment:
Only when valid, the output value should be updated.
So better do the copy to the array here...